src/const.ts

```diff
@@ -38,12 +38,21 @@ export const LEGALDOCUMENT = {
 }
 
 export const RESPONSIVE_SIZES = {
-  MOBILE_SMALL: 320,
-  MOBILE: 500,
-  MOBILE_LARGE: 532,
-  TABLET: 720,
-  TABLET_LARGE: 866,
-  WEB_SMALL: 1024,
+  // PX SIZES:
+  // MOBILE_SMALL: 320,
+  // MOBILE: 500,
+  // MOBILE_LARGE: 532,
+  // TABLET: 720,
+  // TABLET_LARGE: 866,
+  // WEB_SMALL: 1024,
+
+  // REM SIZES:
+  MOBILE_SMALL: 20,
+  MOBILE: 31.25,
+  MOBILE_LARGE: 33.25,
+  TABLET: 45,
+  TABLET_LARGE: 54.125,
+  WEB_SMALL: 64,
```

Comment:
Curiosity: how do you get from `px` screen sizes to `em`?

Reply:
```
1rem = 16px
Yrem = Xpx
```
or did you do something more interesting, @W3stside ?